src/api/routes/transcripts.ts

```diff
@@ -52,8 +54,37 @@ const list = procedure
   return g.transcripts;
 });
 
+const getNameMap = procedure
+  .use(authUser())
+  .input(z.string())
```

Comment:
use `.input(z.object({ transcriptId: z.string() }})`. I think for newer code we should always specify the name of the parameter, only if there is only one parameter.